### PR TITLE
Tutor manager branch

### PR DESCRIPTION
**Pull request by ebriuspythagoras**
Done: IFileManager, FileManager (for storing and modification data in file), TutorManager (CRUD operations), fixed bugs

### CODE REVIEW

File: Classes/Tutor.cs [hunk 1]
```diff
@@ -18,25 +22,10 @@ public Tutor()
...
-        public string ToTeachLesson() //урок по умолчанию 1 час
-        {
-            Seniority += (int)LessonDuration.OneHour;
-            return Finish_str;
-        }
-
         public override string ToString()
         {
-            return "FirstName: " + Info.FirstName + ", LastName: " + Info.LastName + ", ID: " + ID + ", Age: " + Age + ", Seniority: " + Seniority;
+            return Id.ToString() + " " + FirstName + " " + LastName + " " + Birthday + " " + Age.ToString() + " " + Seniority.ToString();
```
Comment (Vadim211) [inline Classes/Tutor.cs]: Would be good to do it in template string. Much easier to read

File: Managers/FileManager.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+using Newtonsoft.Json.Serialization;
+
+namespace ThirdTeam_Study.Managers
+{
+    public class FileManager 
+    {
+        public string FilePath { get; }
+        public readonly JsonSerializerSettings settings;
+        
+        public FileManager(string path = "/Users/volodimir/ThirdTeam_Study/Files/Tutor.json")
```
Comment (Vadim211) [inline Managers/FileManager.cs]: This will broke code on any other computer except yours

File: Managers/FileManager.cs [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                foreach (Tutor tutor in list)
+                    json_list.Add(JsonConvert.SerializeObject(tutor, settings));
+
+                if (!File.Exists(FilePath)) File.Create(FilePath).Close();
+
+                foreach (string json in json_list)
+                    File.WriteAllText(FilePath, ToParceJSON(json));
+                return true;
+            }
+            catch (Exception) { return false; }
```
Comment (Vadim211) [inline Managers/FileManager.cs]: Add some exception error message

File: Managers/FileManager.cs [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        {
+            if (obj == null) throw new NullReferenceException();
+            try
+            {            
+                string json_str = JsonConvert.SerializeObject(obj,settings);
+                if (!File.Exists(FilePath)) File.Create(FilePath).Close();
+                File.WriteAllText(FilePath, ToParceJSON(json_str));
+                return true;
+            }
+            catch (Exception) { return false; }
```
Comment (Vadim211) [inline Managers/FileManager.cs]: Add some exception error message

**Review by Vadim211 — CHANGES_REQUESTED** [review]

File: Managers/FileManager.cs (hunk 4 above)
Comment (ebriuspythagoras) [reply Managers/FileManager.cs]: I have done

File: Managers/FileManager.cs (hunk 3 above)
Comment (ebriuspythagoras) [reply Managers/FileManager.cs]: I have done

File: Managers/FileManager.cs (hunk 2 above)
Comment (ebriuspythagoras) [reply Managers/FileManager.cs]: I have fixed it, so it will work correctly

File: Classes/Tutor.cs (hunk 1 above)
Comment (ebriuspythagoras) [reply Classes/Tutor.cs]: I have done

**Review by IlliaKrasheninnikov — APPROVED** [review]

**Pull request merged**